site/js/components/hero-animated.js

```diff
@@ -25,22 +25,34 @@ export class HeroAnimated {
   constructor(element) {
     // Create a set of constants to be reused throughout the class.
     const constants = {
-      wrapperSelector: '.hero-animated__animation',
-      animationItemClass: 'hero-animated__animation-item',
-      animationDelay: 2000,
-      animationName: 'Hero',
+      animation: {
+        delay: 2000,
```

Comment:
I know we had this delay in the original here, I'm not sure we need it now, though.